api/src/trace/NonRecordingSpan.ts

```diff
@@ -52,6 +53,14 @@ export class NonRecordingSpan implements Span {
     return this;
   }
 
+  addLink(_link: Link): this {
```

Comment:
The coverage check is failing as these are not covered by tests. I think there's a test for the NonRecordingSpan where you could add calls to`addLink()` prevent this. 

(green check mark usually also makes the PR more visible to reviewers)